src/core/templates/buttons.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="buttons-display">
         <div class="buttons-marking-icon icon display-none">
         </div>
-        <div class="buttons-display-inner">
+        <div class="buttons-display-inner" aria-live="rude">
```

Comment:
it also doesn't seem like there's a guarantee it'll read out using aria-live... always a rock and a hard place, at least when it's hard update and available to tab over the potential for it to read is always there. perhaps adding aria-live to the label at the bottom of the component is in order? so we get both if the aria-live is reading.